UnityProject/Assets/Scripts/Managers/CentComm.cs

```diff
@@ -227,6 +202,15 @@ public void MakeCommandReport(string text, UpdateSound type)
 			_ = SoundManager.PlayNetworked(SingletonSOSounds.Instance.AnnouncementCommandReport);
 		}
 
+		/// <summary>
+		/// Makes a written report that will spawn at all comms consoles. Must be called on server.
+		/// </summary>
+		/// <param name="text">String that will be the report body</param>
+		public void MakeQuietCommandReport(string text)
```

Comment:
what's the need of this method?

Reply:
It's all in the name; it makes a quiet command report. Normally, the command report makes a loud, global sound when one is printed, and puts in the chat that there is an incoming message from centcomm. I added this version to allow for one to be printed without notifying the entire station a sheet of paper was printed out.

Reply:
add a bool parameter with default value to the normal method instead

Reply:
good point